babel.config.js

```diff
@@ -26,11 +26,7 @@ module.exports = {
       },
     ],
   ],
-  plugins: [
-    '@babel/plugin-proposal-class-properties',
-    'react-hot-loader/babel',
```

Comment:
It was causing the build to break at some point, I don't recall the exact problem, but since we also have `react-hot-loader` installed and hot-reloading seems to work without `react-hot-loader/babel`, I decided just to ditch the latter as redundant instead of trying to fix the immediate problem.